man/examples-code-check.c

```diff
@@ -85,19 +152,21 @@ int main(int argc, char* argv[])
     if (!strncmp(pdir_ent->d_name, "coap_", sizeof ("coap_")-1) &&
         strstr (pdir_ent->d_name, ".txt.in")) {
       FILE*  fp;
-      int  skip = 1;
-      int  in_examples = 0;
+      int skip = 1;
+      int in_examples = 0;
       int in_synopsis = 0;
-      int  count = 1;
-      char  keep_line[1024] = {0};
-      FILE*  fpcode = NULL;
-      FILE*  fpheader = NULL;
-      char  file_name[300];
+      int count = 1;
+      char keep_line[1024] = {0};
+      FILE* fpcode = NULL;
+      FILE* fpheader = NULL;
+      char file_name[300];
       int is_void_func = 0;
       int is_number_func = 0;
       int is_inline_func = 0;
-      int is_struct = 0;
+      int is_struct_func = 0;
+      int is_ptr_func = 0;
```

Comment:
`is_ptr_func` is set but never read from, therefore gcc complains:

```
examples-code-check.c:167:11: warning: variable ‘is_ptr_func’ set but not used [-Wunused-but-set-variable]
  167 |       int is_ptr_func = 0;
      |           ^~~~~~~~~~~
```

Reply:
My (older) gcc was not complaining so missed this - sorry.  variable now removed and code pushed.